### PR TITLE
add amber TI energy output files for testing

### LINKED ISSUE #5: include Amber FEP test data
In order to implement a parser for Amber FEP alchemistry/alchemlyb#10 we need to have a test data set. 

alchemistry/alchemlyb#10 has a zip file from Ross Walker attached, which would need to be evaluated for its suitability.

### PR DESCRIPTION
**Pull request by shuail**
Fixes #5.

1.  add amber TI output files to test the alchemlyb
2. modify the MANIFEST.in file to include the .out format (amber output file)

### CODE REVIEW

**Comment by orbeckst** [comment]
I updated your comment to reference #5. I will review, hopefully today.

( @dotsdl just in case you have a spare moment and want to do something completely different: I'd be more than happy to have your input, too )

**Comment by orbeckst** [comment]
I am waiting on PR #7 to be approved. I will then ask you to pick a license for this dataset from [recommended open licenses](http://opendefinition.org/licenses/#recommended-conformant-licenses) (preferrably CC0 or CC-BY).

File: src/alchemtest/amber/simplesolvated/descr.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Data Set Characteristics:
+    :Number of Legs: 2 (charge, vdw)
+    :Number of Windows: 5 for charge, 12 for vdw
+    :Length of Windows: 1ns
+    :System Size: 5979 atoms
+    :Temperature: 300 K
+    :Pressure: 1 bar
+    :Alchemical Pathway: (charge + vdw) in ligand 1 --> (charge + vdw) in ligand 2, charge and vdw are running in parellel, soft core is used in vdw
+    :Experimental Free Energy difference: N/A 
+
```
Comment (orbeckst) [inline src/alchemtest/amber/simplesolvated/descr.rst]: remove empty line (I know, you copied from the GMX dataset... that line did not serve any purpose there either and I took it out in PR #7)

File: src/alchemtest/amber/simplesolvated/descr.rst [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    :Missing Values: None
+    :Date: Oct 2017
+    :Donor: Silicon Therapeutics
```
Comment (orbeckst) [inline src/alchemtest/amber/simplesolvated/descr.rst]: **IMPORTANT**: add a line
```reST
:License: `NAME <URL>`_
```
that specifies the license that the dataset is released under. See [Contributing: Licenses](https://github.com/alchemistry/alchemtest/wiki/contributing#licensing). Specifically, pick a license from the list [recommended conformant licenses](http://opendefinition.org/licenses/#recommended-conformant-licenses) (with CC0 or CC-BY preferred). You might have to check with your legal department. 

In PR #7 I updated the GMX dataset and you can have a look at it to see what I did there.

Comment (shuail) [reply src/alchemtest/amber/simplesolvated/descr.rst]: Yes, will consult with the company and should get it back soon.

File: src/alchemtest/amber/simplesolvated/descr.rst (hunk 1 above)
Comment (shuail) [reply src/alchemtest/amber/simplesolvated/descr.rst]: Got it, will change this together with other comments once I get the license info finalized.

**Comment by shuail** [comment]
@orbeckst just updated the license info and amber info under docs folder according to the previous comments.

File: src/alchemtest/amber/simplesolvated/descr.rst [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    :Length of Windows: 1ns
+    :System Size: 5979 atoms
+    :Temperature: 300 K
+    :Pressure: 1 bar
+    :Alchemical Pathway: (charge + vdw) in ligand 1 --> (charge + vdw) in ligand 2, charge and vdw are running in parellel, soft core is used in vdw
+    :Experimental Free Energy difference: N/A 
+    :Missing Values: None
+    :Date: Oct 2017
+    :Donor: Silicon Therapeutics 
+    :License: `CC0 <https://creativecommons.org/publicdomain/zero/1.0/>`_ Public Domain Dedication
```
Comment (orbeckst) [inline src/alchemtest/amber/simplesolvated/descr.rst]: awesome!

**Comment by orbeckst** [comment]
Thanks @shuail , merged. (I squashed all your commits into a single one because in this case all commits belonged together and it gets rid of micro-commits such as "fixed typos".)

**Pull request merged**